.changeset/gorgeous-garlics-exist.md

```diff
@@ -0,0 +1,7 @@
+---
+"@wso2is/admin.server-configurations.v1": patch
+"@wso2is/admin.core.v1": patch
+"@wso2is/i18n": patch
```

Comment:
We don't update i18n module in this PR, do we?

since i18n is inside console, let's patch bump console instead

Reply:
Fixed with: https://github.com/wso2/identity-apps/pull/6680/commits/a938e76e6c9fbc179ee78185b96e953c5b2a0f3c